flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  description = "SQLite/LuaJIT binding for lua and neovim";
+
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+
+  outputs = { self, nixpkgs }: let
+      system = "x86_64-linux";
```

Comment:
it would be nice if other systems are support like darwin and arm, maybe use flake utils?